backend/api/v1/routers/migrations.py

```
@@ -0,0 +1,340 @@
+        migration = await service.get_migration(migration_id)
+        if not migration:
+            raise HTTPException(status_code=404, detail="Migration not found")
+        
+        if migration.is_running():
+            raise HTTPException(
+                status_code=400, 
+                detail="Cannot delete running migration. Cancel it first."
+            )
+        
+        # Use repository directly for deletion
+        from ....infrastructure.database.config import get_db_session
```

Flake8 (7.2.0)

[error] redefinition of unused 'get_db_session' from line 12

(F811)